packages/flutter/lib/src/cupertino/action_sheet.dart

```diff
@@ -30,6 +30,20 @@ const TextStyle _kActionSheetContentStyle = TextStyle(
   textBaseline: TextBaseline.alphabetic,
 );
 
+// This decoration is applied to the blurred backdrop to lighten the blurred
+// image. Brightening is done to counteract the dark modal barrier that
+// appears behind the alert. The overlay blend mode does the brightening.
+// The white color doesn't paint any white, it's just the basis for the
+// overlay blend mode.
+//
+// The decoration is needed for `CupertinoActionSheet` to not appear excessively
+// transparent when placed above a PlatformView. See
+// https://github.com/flutter/flutter/issues/50183.
+const BoxDecoration _kAlertBlurOverlayDecoration = BoxDecoration(
```

Comment:
We had this before there was dark mode. Now we may need 2 to handle the right visual fidelity.